WooCommerce/Classes/Blaze/BlazeEligibilityChecker.swift

```diff
@@ -38,6 +38,9 @@ private extension BlazeEligibilityChecker {
         guard site.isAdmin && site.canBlaze else {
             return false
         }
+        guard site.isJetpackConnected && site.isJetpackThePluginInstalled else {
```

Comment:
Nit: It would be helpful to add a comment here explaining the need for a non-JCP site with a reference to the decision.